### ISSUE TITLE
Error: HttpError: Not Found

### ISSUE THREAD

**Issue by brianpham** [fjogeleit/yaml-update-action#112]
I am seeing an error called HttpError: Not Found, but not exactly sure why that is. Any idea what might be going on here? 

`Run fjogeleit/yaml-update-action@master
  with:
    valueFile: k8s/deployment.yaml
    propertyPath: metadata.name
    value: testing
    branch: deployment/bpham
    targetBranch: main
    createPR: true
    message: Update image tag successful
    commitChange: true
    token: ***
    repository: dialpad/dev-actions
    workDir: .
##[debug]FilePath: /home/runner/work/dev-actions/dev-actions/k8s/deployment.yaml, Parameter: {"cwd":"/home/runner/work/dev-actions/dev-actions","workDir":".","valueFile":"k8s/deployment.yaml"}
##[debug]Parsed JSON: {"apiVersion":"apps/v1","kind":"Deployment","metadata":{"name":"test-agent"},"spec":{"template":{"spec":{"containers":[{"image":"fatline.io/session-controller:v2.11","name":"session-controller"},{"image":"fatline.io/browser-agent:v2.11","name":"browser-agent"}]}}}}
##[debug]Generated updated YAML
##[debug]    
##[debug]apiVersion: apps/v1
##[debug]kind: Deployment
##[debug]metadata:
##[debug]  name: testing
##[debug]spec:
##[debug]  template:
##[debug]    spec:
##[debug]      containers:
##[debug]        - image: fatline.io/session-controller:v2.11
##[debug]          name: session-controller
##[debug]        - image: fatline.io/browser-agent:v2.11
##[debug]          name: browser-agent
##[debug]
##[debug]
Error: HttpError: Not Found
##[debug]Node Action run completed with exit code 1
##[debug]Finishing: Update the image tag`

**Comment by fjogeleit** [comment]
Since it is an HTTP error, I assume that the error is coming from the GitHub REST API. You should check if your token has the permissions to commit to your repository. So it needs read and write permissions.

**Comment by brianpham** [comment]
Yeah, looks my token didn't have the right permissions. Thanks, closing.

**Comment by plzm** [comment]
I can confirm this: I also got HttpError: Not Found.

I only wanted to update a YAML file in the worker, and not commit/PR back to the repo. I set the following action input values:
          updateFile: 'true'
          commitChange: 'false'
          createPR: 'false'

This resolved the HttpError and correctly updated the YAML file locally on the runner, ready for later steps. Leaving this comment here to help others (and read the input defaults...)

**Comment by fjogeleit** [comment]
Hey @plzm thank you for your hint. I add an additional example to the readme for this case. I hope this will help other users with a similar use case.
